.github/workflows/lighthouse.yml

```
@@ -25,4 +25,4 @@ jobs:
         configPath: ./lighthouserc.yml
         runs: 1
       env:
-        SITE_URL: https://dev.simplereport.org/
\ No newline at end of file
+        SITE_URL: https://dev.simplereport.gov/
```

I think this action was added before dev had an authentication layer. I believe we will now need to use demo or training 
```suggestion
        SITE_URL: https://demo.simplereport.gov/
```